guides/v2.3/graphql/reference/quote-add-simple-products.md

```diff
@@ -0,0 +1,195 @@
+---
+group: graphql
+title: addSimpleProductsToCart mutation
+---
+
+Simple products are physical products that do not have variations, such as color, size, or price. The `addSimpleProductsToCart` mutation allows you to add multiple simple products to the cart at the same time, but you cannot add other product types with this mutation. To add a simple product to a cart, you must provide the cart ID, the SKU, and the quantity. You can also optionally provide customizable options.
+
+{:.bs-callout .bs-callout-info}
+The mutation for [adding configurable products]({{page.baseurl}}/graphql/reference/configurable-product.html) is defined in the `ConfigurableProductGraphQl` module.
+
+## Syntax
+
+`mutation: {addSimpleProductsToCart(input: AddSimpleProductsToCartInput): {AddSimpleProductsToCartOutput}}`
+
+## Example usage
+
+These examples show the minimal payload and a payload that includes customizable options.
+
+### Add a simple product to a cart
+
+The following example adds a simple product to a cart. The response contains the entire contents of the customer's cart.
+
+**Request**
+
+```text
+mutation {
+  addSimpleProductsToCart(
+    input: {
+      cart_id: "IeTUiU0oCXjm0uRqGCOuhQ2AuQatogjG",
```

Comment:
`,` can be omitted in the request body according to [Graphql Docs](https://github.com/graphql/graphql-spec/blob/master/spec/Section%202%20--%20Language.md#insignificant-commas). It can improve the clearness of the examples.